examples/docs_snippets/docs_snippets/guides/components/index/34-dg-component-check-defs.txt

```diff
@@ -2,10 +2,7 @@ dg check defs
 
 All components validated successfully.
 
-  warnings.warn(message)
-
 INFO:dagster.builtin:Running dbt command: `dbt parse --quiet`.
 INFO:dagster.builtin:Finished dbt command: `dbt parse --quiet`.
-Using /.../jaffle-platform/.venv/bin/dagster
-uv run dagster definitions validate --log-level warning --log-format colored --workspace /tmp/workspace
+dagster definitions validate --log-level warning --log-format colored --workspace /tmp/workspace
```

Comment:
is this change expected?

Reply:
Yes. The tutorial no longer uses `uv run` at all because both paths use an activated venv.